atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweeperTest.java

```diff
@@ -127,8 +129,13 @@ protected static void setupTables(KeyValueService kvs) {
     }
 
     private static void tearDownTables(KeyValueService kvs) {
-        kvs.getAllTableNames().stream()
-                .forEach(tableRef -> kvs.dropTable(tableRef));
+        Awaitility.await()
+                .timeout(Duration.FIVE_MINUTES)
```

Comment:
It takes about an hour if the build is going to fail. @ashrayjain: for SA, the heartbeat is every second, so maybe  we can reduce the 5 minutes to 1 minute?